src/client/views/select-issuers.ts

```diff
@@ -178,8 +179,8 @@ export class SelectIssuers extends AbstractView {
             <li class="issuer-connect-banner-tn" data-issuer="${issuer}" role="menuitem">
               <div tabindex="0" style="display: flex; align-items: center;">
                 <div class="img-container-tn issuer-icon-tn shimmer-tn" data-image-src="${
-									image ?? ''
```

Comment:
This code change seems to be in every one of your commits. Did you turn off all your auto formatting plugins? 
Until this is resolved, try not to commit changes that aren't necessary to the PR.

On my side, this comes up as a formatting error for ESLint, but is how prettier is formatting it. So it looks like eslint & prettier disagree on this one. We need to add an exception for eslint or prettier to stop this issue. Maybe there is a way to ignore indent in template literals.

Reply:
Thanks, I'll take a look at this. 

I'm using the official VS code Prettier Plugin which seems to reformat the code this way.

<img width="1079" alt="Screen Shot 2023-03-01 at 1 12 53 pm" src="https://user-images.githubusercontent.com/6808817/222026989-80029899-4308-4f59-ae27-4b0871332dbe.png">

Reply:
From running a test just now, it's the official Prettier VScode plugin that is applying this opinionated change to the code. However we did choose to move to using Prettier over ESlint to align with Hu's team's settings (as closely as possible), where I think it best we come back to this issue as a larger review task using this ticket below.

https://smarttokenlabs.atlassian.net/browse/TKS-830

Reply:
Yes but we can't go on with this file being changed every single commit. Let's just rely on Husky for formatting. It runs prettier after eslint, which makes me think this plugin isn't picking up the config file correctly. 

Alternatively, seems we can use a prettier plugin in eslint config to disable eslint formatting: plugin:prettier/recommended but I haven't been able to get it working.